dev/Vagrantfile

```diff
@@ -56,7 +56,6 @@ end
 
 Vagrant.configure("2") do |config|
   attributes = load_settings
-  config.omnibus.chef_version = :latest
```

Comment:
'omnibus' is not a supported attr of `config` anymore. This has no functional effect on dev vm, because we provision using the chef client included in chef server.